pkg/networkservice/chains/nsmgr/suite_test.go

```diff
@@ -603,7 +668,7 @@ const (
 	labelB = "label_b"
 )
 
-func linearNS(count int) *registry.NetworkService {
+func linearNS(count int, twoEmpty, withPassthrough bool) *registry.NetworkService {
```

Comment:
This looks complicated. Please use this inline for the new test.

Reply:
done